bats/tests/registry/creds.bats

```diff
@@ -1,7 +1,19 @@
+setup_file() {
+    rm -f "$HOME/.docker/config.json"
```

Comment:
Please move this to the factory-reset test. Logically it belongs together, it is just that factory-reset doesn't delete this file.

Reply:
Done.